addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -3041,6 +3052,9 @@ void zfEnable()
 
 	delete zf_tDataCollect;
 	zf_tDataCollect = CreateTimer(2.0, timer_datacollect, _, TIMER_REPEAT);
+	
+	delete zf_tTimeProgress;
+	zf_tTimeProgress = CreateTimer(30.0, timer_progress, _, TIMER_REPEAT);
```

Comment:
Ideally should start timer when round starts, and killing current timer when starting new round. g_fTimeProgress also never got reset during it.